### PR TITLE
added slice by step

### PR DESCRIPTION
**Pull request by redraw**
#221

### CODE REVIEW

File: test/test.py [hunk 1]
```diff
@@ -284,6 +284,17 @@ def test_slicing(self):
         past_end = second_long_slice[:1500]
         self.assertTrue(second_long_slice._data == past_end._data)
 
+    def test_slicing_by_step(self):
+        audio = self.seg1[:10000]
+        chunks = audio[:0]
+
+        for chunk in audio[::1000]:
+            self.assertTrue(isinstance(chunk, audio.__class__))
```
Comment (jiaaro) [inline test/test.py]: I think we can safely assert `isinstance(chunk, AudioSegment)`

**Comment by jiaaro** [comment]
Note: Appveyor is having a problem, the tests are erroring out, not failing.

**Comment by jiaaro** [comment]
Crap I didn't mean to close it - sorry!

File: test/test.py (hunk 1 above)
Comment (redraw) [reply test/test.py]: true!..I can wait to commit, to run Appveyor too

Comment (jiaaro) [reply test/test.py]: ok Appveyor is fixed, just pull from pydub master

**Comment by redraw** [comment]
guess you didn't want that merge commit, wasn't sure how to avoid it

**Comment by jiaaro** [comment]
Merge commit is no problem :)

**Pull request merged**